appIcons.js

```diff
@@ -1203,3 +1183,90 @@ function itemShowLabel()  {
         transition: 'easeOutQuad',
     });
 }
+
+const ApplicationOverview = new Lang.Class({
+    Name: 'DashToDock.ApplicationOverview',
+
+    _init: function() {
+        this.isInAppOverview = false;
+        this.originalTriggerSearchFunction = ViewSelector.ViewSelector.prototype._shouldTriggerSearch;
+        this.originalOverviewFunction = Workspace.Workspace.prototype._isOverviewWindow;
+        this.originalThumbnailFunction = WorkspaceThumbnail.WorkspaceThumbnail.prototype._isOverviewWindow;
+        this.hidingId = Main.overview.connect('hidden', Lang.bind(this, this._onOverviewHidden));
+    },
+
+    dissconect: function() {
+        Main.overview.dissconect(this.hidingId);
```

Comment:
You are absolutely right. Always looked strange to me to ;) Fixed it with new commit.  Thank you!
Where did you see the errors? Im currently alway using 
`tail -f /var/log/syslog | grep Gjs-Message`
Is there a better way?